filament/src/Scene.cpp

```diff
@@ -364,6 +364,12 @@ void FScene::remove(Entity entity) {
     mEntities.erase(entity);
 }
 
+void FScene::removeEntities(const Entity* entities, size_t count) {
+    for (size_t i = 0; i < count; ++i, ++entities) {
+        mEntities.erase(*entities);
```

Comment:
maybe this should just call `remove(Entity)`?

Reply:
Done